modules/backend/formwidgets/iconpicker/assets/src/app.js

```diff
@@ -0,0 +1,12 @@
+import { createApp } from 'vue'
+import App from './components/App.vue'
+
+window.addEventListener('load', () => {
+    const elements = document.querySelectorAll("[data-control='iconpicker']:not([data-v-app])");
+    elements.forEach(element => {
+        createApp(App, {
+            ...element.dataset,
+            fontLibraries: JSON.parse(element.parentElement.querySelector("script[type='application/json']").textContent)
```

Comment:
Font libraries can be loaded over AJAX to avoid large quantities of data being duplicated unnecessarily on page load